test/smoke/src/helpers/setupEnvironmentHelper.ts

```diff
@@ -23,6 +23,7 @@ const isInsiders = version === "insiders";
 const downloadPlatform = (process.platform === "darwin") ? "darwin" : process.platform === "win32" ? "win32-archive" : "linux-x64";
 const androidEmulatorPort = 5554;
 export const androidEmulatorName = `emulator-${androidEmulatorPort}`;
+export const androidEmulatorVersion = "9";
```

Comment:
Please avoid magic constants like "9", "emulator-5554". I'd recommend you to create constants class that contains such constants with their descriptions in comments.

Reply:
Agree.